packages/itwinui-react/src/core/utils/components/InputFlexContainer.tsx

```diff
@@ -6,6 +6,8 @@ import React from 'react';
 import cx from 'classnames';
 import type { PolymorphicForwardRefComponent } from '../props.js';
 import { Box } from './Box.js';
+import { Icon, type IconProps } from '../../Icon/Icon.js';
+import { IconButton, type IconButtonProps } from '../../Buttons/IconButton.js';
```

Comment:
we should use `import type` for better compatibility (https://github.com/iTwin/iTwinUI-react/pull/996)

Reply:
Done (https://github.com/iTwin/iTwinUI/pull/1881/commits/bdbd7db0202214c08d014ff5722a96e8e1cff4fe).

Reply:
Missed one https://github.com/iTwin/iTwinUI/blob/30c62de52fec366b97eaecb0ee2bec2d4b409fed/packages/itwinui-react/src/core/InputWithDecorations/InputWithDecorations.tsx#L14

Reply:
Ohh, fixed (https://github.com/iTwin/iTwinUI/pull/1881/commits/d9466bae103ebb3f28cda5506122d29886730a92). Also double checked for other such changes in this PR and found none.